.taskcluster.yml

```diff
@@ -0,0 +1,272 @@
+# yamllint disable rule:line-length
+# This file is rendered via JSON-e by
+# - github events - https://github.com/taskcluster/taskcluster/tree/main/services/github
+# - cron tasks - https://hg.mozilla.org/ci/ci-admin/file/default/build-decision/
+# - action tasks - taskcluster/taskgraph/actions/registry.py
+---
+version: 1
+reporting: checks-v1
+autoCancelPreviousChecks: true
+policy:
+    pullRequests: public
+tasks:
+    - $let:
+          trustDomain: "mozilla"
+          level: "1"
+          ownerEmail:
+              $switch:
+                  'tasks_for == "github-push"': '${event.pusher.email}'
+                  'tasks_for[:19] == "github-pull-request"': '${event.pull_request.user.login}@users.noreply.github.com'
+                  'tasks_for in ["cron", "action"]': '${tasks_for}@noreply.mozilla.org'
+          baseRepoUrl:
+              $switch:
+                  'tasks_for == "github-push"': '${event.repository.html_url}'
+                  'tasks_for[:19] == "github-pull-request"': '${event.pull_request.base.repo.html_url}'
+                  'tasks_for in ["cron", "action"]': '${repository.url}'
+          repoUrl:
```

Comment:
Any issues with some of these being camelCase (`ownerEmail` L16, `baseRepoUrl` L21, or `repoUrl` L26) vs underscored (`head_branch` L36, `base_ref` L42, etc)?

Reply:
These were generated by `taskgraph init` and/or copied from https://github.com/mozilla-releng/firefox-ci-playground, so this is a code convention decision made by the creators of taskgraph and taskcluster.